guides/common/modules/proc_exporting-a-content-view-version-incrementally.adoc

```diff
@@ -7,7 +7,7 @@ The size of the exported Content View versions depends on the number of products
 endif::[]
 Content View versions that have multiple {RHEL} trees can occupy several gigabytes of space on {ProjectServer}.
 
-In such cases, you can use *Incremental Export* to export only pieces of content that changed since the previous export.
+In such cases, you can create an incremental export which contains only pieces of content that changed since the previous export.
```

Comment:
```suggestion
In such cases, you can create an incremental export which contains only pieces of content that got changed since the previous export.
```

Reply:
Same as above, I think it's probably best to leave as is so it doesn't sound past tense with `got`.